src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -100,7 +100,7 @@ const ChatItemTargetText = ({
   };
 
   return (
-    <View style={styles.chatContainer}>
+    <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

As mentioned above, using negative margins can lead to overlapping elements and should be avoided. Consider revising your layout strategy to avoid the need for negative margins.
```diff
- <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
+ <View style={styles.chatContainer}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->